bin/compile

```diff
@@ -6,32 +6,31 @@ $stdout.sync = true
 BUILD_DIR = ARGV[0]
 CACHE_DIR = ARGV[1]
 ARCHIVE_NAME = "freetds-patched"
+INSTALL_DIR = "#{BUILD_DIR}/vendor/freetds"
 
 puts "-----> FreeTDS building in #{BUILD_DIR}"
+puts `mkdir -p #{INSTALL_DIR}`
 
 puts "-----> Fetching & Extracting FreeTDS"
-puts `mkdir -p #{CACHE_DIR}`
-puts `curl ftp://ftp.freetds.org/pub/freetds/stable/#{ARCHIVE_NAME}.tar.gz -o - | tar -xz -C #{CACHE_DIR} -f -`
+puts `curl ftp://ftp.freetds.org/pub/freetds/stable/#{ARCHIVE_NAME}.tar.gz -o - | tar -xz -C #{INSTALL_DIR} -f -`
 
-TARGET_DIR = `ls #{CACHE_DIR} | grep 'freetds-[0-9]\.'`.chomp
+TARGET_DIR = `ls #{INSTALL_DIR} | grep 'freetds-[0-9]\.'`.chomp
 puts "installing #{TARGET_DIR}"
 
-puts "-----> Configurating FreeTDS"
-puts `mkdir -p #{BUILD_DIR}/vendor/freetds`
-puts `cd #{CACHE_DIR}/#{TARGET_DIR} && ./configure --prefix=#{BUILD_DIR}/vendor/freetds --with-libiconv-prefix=DIR --with-openssl=DIR --with-tdsver=7.3`
+puts "-----> Configuring FreeTDS"
+puts `cd #{INSTALL_DIR}/#{TARGET_DIR} && ./configure --prefix=#{INSTALL_DIR} --with-libiconv-prefix=DIR --with-openssl=DIR --with-tdsver=7.3`
 
 puts "-----> Building FreeTDS"
-puts `cd #{CACHE_DIR}/#{TARGET_DIR} && make`
+puts `cd #{INSTALL_DIR}/#{TARGET_DIR} && make`
 
-puts "-----> Installing FreeTDS into #{BUILD_DIR}/vendor/freetds"
-puts `mkdir -p #{BUILD_DIR}/vendor/freetds`
-puts `cd #{CACHE_DIR}/#{TARGET_DIR} && make install`
+puts "-----> Installing FreeTDS into #{INSTALL_DIR}"
+puts `cd #{INSTALL_DIR}/#{TARGET_DIR} && make install`
 
-puts "-----> Copying FreeTDS into /app/vendor/freetds"
```

Comment:
I think the lines below here need to stay (with `BUILD_DIR` replaced with `INSTALL_DIR`)?

As I understand it, everything in `BUILD_DIR` will not be accessible to the app when running inside a Heroku dyno, so I think it's still important to copy it to the '/app' directory.